docs/pages/application-access/guides/api-access.mdx

```diff
@@ -5,8 +5,9 @@ description: How to access REST APIs with Teleport Application Access.
 
 # Application API Access
 
-Teleport Application Access can be used to access applications' (REST or Teleport's own gRPC) APIs with
-tools like [curl](https://man7.org/linux/man-pages/man1/curl.1.html) or Postman.
+Teleport Application Access can be used to access HTTP applications' APIs with
```

Comment:
gRPC would work with TCP app access, right?

Reply:
Yeah, you thinking I should mention something like, "For non-HTTP API purposed (like gRPC) please use Application Access TCP?"

Reply:
Yes